initfiles/componentfiles/configxml/buildsetCC.xml.in

```diff
@@ -330,6 +330,12 @@
     </BuildSet>
     <BuildSet deployable="no"
              installSet="deploy_map.xml"
+             name="cassandraloggingagent"
```

Comment:
Make sure we're not adding these components to the Configmanager UI if the entire UI isn't implemented yet.